backend/service/k8s/configmap.go

```diff
@@ -2,12 +2,47 @@ package k8s
 
 import (
 	"context"
+	"fmt"
 
 	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	k8sapiv1 "github.com/lyft/clutch/backend/api/k8s/v1"
 )
 
+func (s *svc) DescribeConfigMap(ctx context.Context, clientset, cluster, namespace, name string) (*k8sapiv1.ConfigMap, error) {
+	cs, err := s.manager.GetK8sClientset(ctx, clientset, cluster, namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	configMapList, err := cs.CoreV1().ConfigMaps(cs.Namespace()).List(ctx, metav1.ListOptions{
+		FieldSelector: "metadata.name=" + name,
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(configMapList.Items) == 1 {
+		return protoForConfigMap(cs.Cluster(), &configMapList.Items[0]), nil
+	} else if len(configMapList.Items) > 1 {
+		return nil, fmt.Errorf("Located multiple configMaps")
```

Comment:
Maybe say `"Located multiple configMaps with name %s", name`? 

Technically we should never hit this case cause kubernetes would never allow more than once configmap with the same name and we filter on the name in the list command

Reply:
All the other resources have the same error message, so I guess its okay to leave it as is :) upto you :)

Reply:
added in https://github.com/lyft/clutch/pull/930/commits/147f0d0b6cd71926d777622693c9b8e90e5b7349